### ISSUE TITLE
Google Drive and pdf zoom flicker - when using open in new window

### ISSUE THREAD

**Issue by nmat** [wavebox/waveboxapp#393]
* Which version of Wavebox are you using? [e.g. 3.0.0]
3.3.3beta

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Windows 10

* Description
When I open a pdf file for reading a book or similar then zoom in to the window still keeping the preview in google drive it zooms both views.

If I then go somewhere else I notice a flicker at some points when I think there is a background refresh or something that causes problems for the eyes since you would most likely expect the zoom to be for that view only even if it is in a window on its own.

Can someone please see if they have the same type of issue?

**Comment by Thomas101** [comment]
Okay, yeah I see the same. It looks like the content windows are sharing the same zoom setting as the accounts. This shouldn't be the case. I'll look to see what's going on here

**Comment by nmat** [comment]
perfect thank you! yes reading pdf's etc would benifit from different zoom in different windows for sure. (just a example) good that you could reproduce it.

**Comment by Thomas101** [comment]
I've been (finally) taking a look into this. What's happening behind the scenes, is when you open the new window it holds a reference to its parent as the opener and opens both windows on the same site instance. This means that you change the zoom level on one, it changes across the two windows.

This is the same behaviour exhibited by Chrome and firefox...

![dec-15-2017 11-08-54_2](https://user-images.githubusercontent.com/103586/34039796-f08d1ef6-e188-11e7-8e52-50d5dc4d5f6f.gif)

So the initial changing, I think is the desired behaviour and we should probably keep to match the browser experience.

There is another issue here though. When the zoom gets changed from the popout window, it doesn't write that change back to the database. Which means at some point (either on navigation or re-launch) the opening window in the main UI will flit back to whatever it was.

At the moment the data that the popout window has is just a read-only copy which is piped across on launch. It doesn't really have a way to write that data back. We've got some plans to add support for writing data back, but this falls under a bigger set of features we're looking at adding soon.

I'm going to keep the issue open (unless you're happy to close it), but remove it off the milestone - as the fix falls under a bunch of other things we're looking at doing and for the time-being it's just being moved from one milestone to the next.

**Comment by nmat** [comment]
Yeah that sounds like a proper way to do it. The initial zoom is still suppose to be there so it makes sense in that case.

Can keep the ticket open as reminder but its not really something "important" at the moment. I think that the level of when it can be annoying is when viewing pdfs that has some sort of its own zoom "at the bottom" you see there. But I am perfectly happy. :)

**Comment by Tahini-123** [comment]
I dont know if it's related by I am experiencing the same and it's making editing Google Docs impossible as the zoom is adjusted all the time. Please have a look at these short clips
[https://photos.app.goo.gl/VQb8abjMyvL9FNIQ2](url)
[https://photos.app.goo.gl/ijzaNIi45nPeLAYv1](url). 

Can we get it to stop? Or at least is there a way for me to make it stop?

**Comment by Thomas101** [comment]
@Tahini-123 I think that may be a different issue. So we can check, would you be able to try the following...

1. When in your Google Drive tab in Wavebox, go to the View menu and then Reset Zoom
2. Open a document
3. Do you still see the behaviour?

**Comment by Thomas101** [comment]
Unblocked by https://github.com/wavebox/waveboxapp/commit/b0599061a502d21db72d3e63f73e0b9f0961a495

**Comment by Thomas101** [comment]
Hi, we’ve just released this to the beta channel of Wavebox in version 3.10.4. If you want to give it a try there are some instructions on [how to use the beta channel here](https://wavebox.io/kb/beta-channel). Thanks

**Comment by Thomas101** [comment]
Hi, this has just been released in the latest version of Wavebox. You can get the update by using the Application/File menu and clicking `Check for Update` or by downloading the latest version from our [download page](https://wavebox.io/download). Thanks!
